developer_setup.md

```diff
@@ -83,17 +83,34 @@ As per 20+, with the following changes:
   brew install git
   brew install memcached
   brew install postgresql
+  brew install graphviz
   ```
 
+If `bundle install` (when running `bin/setup` below) fails in installing the `eventmachine` gem,
```

Comment:
While I agree with you, I left it in the OS/X section as it is OS/X specific. The `bin/setup` step is generic.

But then it could be "If bin/setup fails on OS/X on installing the evenmachine gem, run the following command and then retry bundle install:"